metadata.md

```diff
@@ -1,35 +1,99 @@
+- **sampleMeanNormal**: Sample mean normalizing Ct/Cq.
+- **sampleVarianceNormal**: Sample variance normalizing Ct/Cq.
+
+## Site (wastewater_site.csv) <span id="site"><span>
+
+Site of SARS-CoV-2 wastewater sampling, including how the sample was collected and stored. Each SARS-CoV-2 test result corresponds to one site.
+
+- **siteID**:	Unique identifier for the location where wastewater sample was taken. `locationID` is the same in `wastewater_covid-19.csv`. (Primary Key)
+- **siteName**:	Name corresponding to `siteID`. Location name could be a treatment plant, campus, institution or sewer location, etc. 
+- **siteDescription**: Description of wastewater site (city, building, street, etc., to identify location sampled).
+- **laboratoryID**: Unique identification of laboratory that is perform testing at the site.
+- **siteType**: Type of sample collected at site. 
+  - `sludge`: Primary clarified sludge
+  - `rawCollector`: Raw wastewater (in collector system)
+  - `rawPostGrit`: Raw wastewater (post-grit)
+  - `other`: Other type of sample. Add description to `sampleTypeOther`.
+- **methodCollection**: method of sample collection.
+  - `24hq1h`: 24-hour composite, every 1 hr
+  - `24hq4h`: 24-hour composite, every 4 hr
+  - `24hq6h`: 24-hour composite, every 6 hr
+  - `grab`:   Grab sample
+  - `other`:  Other method collection type. Add description to `methodCollectOther`.
+- **methodCollectOther**: Description of other type of method of sample collection. see `methodCollection`.
+- **sampleFraction**: Faction of the sample that is analyzed.
+  - `liquid`: Liquid fraction
+  - `solid`:  Solid fraction
+  - `mixed`:  Mixed/homogenized sample
+- **geoLat**: Site geographical location, latitude in decimal coordinates, ie.: (45.424721)
+- **geoLong**: Site geographical location, longitudein decimal coordinates, ie.: (-75.695000)
+- **storageTemp**: Storage conditions/temperature of sample (degree Celcius).
+- **contactName**: Person to contact about data collection at the site.
+- **contactEmail**: Email addrss for `contactName`.
+- **conatctPhone**: Phone number for `contactName`.
+- **catchmentPop**: Approximate population size of catchment area corresponding to site. The number of people resprested in the wastewater sample.
+- **notes**: Any addtional notes.
+
+## Laboratory (wastewater_lab.csv) <span id="lab"><span>
+
+Laboratory that performs SARS-CoV-2 wastewater testing at one or more sites.
+
+- **laboratoryID**: Unique identifier for the laboratory. (Primary Key)
+- **laboraotryName**: Name corresponding to `siteID`.
+- **contactName**: Contact person or group.
+- **contactEmail**: Contact e-mail address.
+- **contactPhone**: Contact phone number.
+- **date**: Date information was provided or updated.
+
+
+## Entity Relationship Diagram <span id="erd"><span>
+
+![](img/ERD.svg)
+
+Comment on the ERD in [Lucidcharts](https://lucid.app/invitations/accept/781822fc-6ac5-4aa7-9023-323fd4b6b04f)
+
+## Changelog
```

Comment:
Shouldn't this be in the [README](https://github.com/Big-Life-Lab/covid-19-wastewater/blob/metadata/README.md#changelog)?